content/en/tlsalpnrevocation.md

```diff
@@ -12,7 +12,16 @@ This page hosts the list of certificates affected by the TLS-ALPN-01 Incident. Y
 
 We have sent notification emails to affected subscribers who have registered a valid email address; if you received an email, you have at least one affected certificate. You may not have received an email if you did not provide an email address when registering your ACME account, if you unsubscribed from Let's Encrypt [email notifications](https://letsencrypt.org/docs/expiration-emails/) previously, or if the email was caught by a spam filter.
 
-If you are unsure whether your hostname is affected, we are also providing a list of all affected certificate serial numbers, along with their associated account ID numbers and all DNS Names which appear on the certificate.
```

Comment:
How about keeping some of this and shortening you're text:

If you are unsure whether your hostname is affected, we are providing a list of all affected serial numbers, along with their associated account ID numbers and all DNS Names which appear on the certificate. You can use our [web tool](https://tls-alpn-check.letsencrypt.org) to search the data set and see if you are affected. Tool is also accessible command line interface like so:
< what you have>
Alternatively, the dataset is available for download and local searching.